lib/Models/ImageryLayerCatalogItem.js

```diff
@@ -151,6 +151,20 @@ var ImageryLayerCatalogItem = function(terria) {
    */
   this.dontChartAlone = false;
 
+  /**
+   * Gets or sets the color of how the dataset will displayed on the chart
+   * @type {String}
+   * @default '#44C99F'
+   */
+  this.chartColor = this.chartColor ? this.chartColor : "#44C99F";
```

Comment:
`this.chartColor` can't be defined already on this line because it is created for the first time on this line.